lib/handlers/authorize-handler.js

```diff
@@ -28,7 +28,7 @@ var url = require('url');
 
 var responseTypes = {
   code: require('../response-types/code-response-type'),
-  token: require('../response-types/token-response-type')
+  //token: require('../response-types/token-response-type')
```

Comment:
Is there a chance, this is ever needed again? Because otherwise I would remove it completely. Not just comment out.

Looking at:
```
if (!_.has(responseTypes, responseType)) { ... }
```

There is IMHO an easy chance if introducing a bug here.

Besides: Thanks so much fox fixing this issue. 👍